modules/docs/src/main/mdoc/docs/03-Connecting.md

```diff
@@ -32,18 +32,17 @@ This is a perfectly respectable **doobie** program, but we can't run it as-is; w
 ```scala mdoc:silent
 import doobie.util.ExecutionContexts
 
-// We need a ContextShift[IO] before we can construct a Transactor[IO]. The passed ExecutionContext
-// is where nonblocking operations will be executed. For testing here we're using a synchronous EC.
-implicit val cs = IO.contextShift(ExecutionContexts.synchronous)
+// This is just for testing. Consider using cats.effect.IOApp instead of calling calling 
```

Comment:
This file an bunch of other `.md` files with the same comment have a typo `calling calling`

Reply:
Good catch. Fixed.